README.md

```diff
@@ -57,6 +57,8 @@ The following are the driver options that can be configured using a DSN or conne
 | `LogLevel` | Severity level for driver logs. | one of `ES_OFF`, `ES_FATAL`, `ES_ERROR`, `ES_INFO`, `ES_DEBUG`, `ES_TRACE`, `ES_ALL` | `ES_WARNING` |
 | `LogOutput` | Location for storing driver logs. | string | WIN: `C:\`, MAC: `/tmp` |
 
+Administrative privileges are required to change the value of logging options.
```

Comment:
Add a bold **NOTE** here to highlight this point.

Reply:
added